src/PluginMediaStreamTrack.swift

```diff
@@ -18,7 +18,11 @@ class PluginMediaStreamTrack : NSObject {
 		if (trackId == nil) {
 			// Handle possible duplicate remote trackId with  janus or short duplicate name
 			// See: https://github.com/cordova-rtc/cordova-plugin-iosrtc/issues/432
-			self.id = rtcMediaStreamTrack.trackId + "_" + UUID().uuidString;
+			if (rtcMediaStreamTrack.trackId.count<36) {
```

Comment:
There is another issue with the stop method the self.eventListener can be nil if one of the participants published a track that was never listened to. So need to add an if (self.eventListener != nil)

Reply:
There is already a PR for that https://github.com/cordova-rtc/cordova-plugin-iosrtc/pull/602 see issue #601 for initial report.